chat settings typing notice and user configuration (#1656)
acterglobal/a3: .changes/811-chat-typing-feature.md

> - [feat.] Chat now supports typing notifications and inform you who's been typing. This setting is enabled by default but can be turned off in App Settings -> Chat Settings.

Reviewer: ```suggestion
- [feat.] Chat now shows typing notifications to inform you who's typing. This setting is enabled by default but can be turned off in App Settings -> Chat Settings.
```
